### PR TITLE
Epsonprinter

### PR DESCRIPTION
**Pull request by ThaStealth**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#23144

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by klaasnicolaas** [comment]
I think that it is better if you put this direct in the `current` branch, because the parent PR is already included in the 0.92 release and online.

File: source/_components/epsonprinter.markdown [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+title: "Epson Printer"
+description: "Instructions on how to integrate Epson Workforce Printer into Home Assistant."
+date: 2019-04-09 16:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: epson.png
+ha_category: Sensor
+ha_release: 0.93
```
Comment (klaasnicolaas) [inline source/_components/epsonprinter.markdown]:
```suggestion
ha_release: 0.92
```

File: source/_components/epsonprinter.markdown [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+ha_iot_class: Local Polling
+redirect_from:
```
Comment (klaasnicolaas) [inline source/_components/epsonprinter.markdown]: You can remove line 14 and 15

File: source/_components/epsonprinter.markdown [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+Assistant.
+
+To add Epson Printer to your installation,
+add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+   - platform: epsonprinter
+     host: 192.168.0.123
```
Comment (klaasnicolaas) [inline source/_components/epsonprinter.markdown]:
```suggestion
     host: IP_ADDRESS
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/epsonprinter.markdown [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+The `epson printer` platform allows you to monitor the ink levels of a Epson Workforce printer from Home
+Assistant.
+
+To add Epson Printer to your installation,
+add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+media_player:
```
Comment (klaasnicolaas) [inline source/_components/epsonprinter.markdown]: Why is it using media_player?

Comment (ThaStealth) [reply source/_components/epsonprinter.markdown]: It isn't, it is is using sensor, changed it

**Comment by klaasnicolaas** [comment]
Ready for merge I think?

**Review by klaasnicolaas — APPROVED** [review]

**Comment by klaasnicolaas** [comment]
When Netifly is ready I will perform the merge

**Comment by SeanPM5** [comment]
I don't think this is correct config. Platform should be `epsonworkforce` and not `epsonprinter` I believe. 

When I had `platform: epsonprinter` and then checked config it gave me invalid config errors about the sensor integration `epsonprinter` not being found. 

Could you double check @ThaStealth ?

This config worked for me:
```
sensor:
  - platform: epsonworkforce
    host: 192.168.1.175
    monitored_conditions:
      - black
      - yellow
      - magenta
      - cyan
      - clean
```

However when I say "worked" i mean it just accepted it as valid and allowed me to restart HA. After restarting it still doesn't seem to be initiating sensor after many retries. 

```
Platform epsonworkforce not ready yet. Retrying in 30 seconds.
Platform epsonworkforce not ready yet. Retrying in 60 seconds.
Platform epsonworkforce not ready yet. Retrying in 90 seconds.
Platform epsonworkforce not ready yet. Retrying in 120 seconds.
Platform epsonworkforce not ready yet. Retrying in 150 seconds.
Platform epsonworkforce not ready yet. Retrying in 180 seconds.
```

**Comment by ThaStealth** [comment]
You are correct, I had to rename the platform and something weird happend with the PR, I guess we merged an old version.

**Comment by klaasnicolaas** [comment]
Just create a new PR then we can fix it 😉

**Pull request merged**